tools/audio/extraction/audio_extract.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+# audio_extract.py
```

Comment:
Why do these files have a name tag? It seems like it would only make it ever-so-slightly harder to rename the files